modules/headless_ui/modules/access_ui/css/icons/settings.svg

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="iso-8859-1"?>
+<!-- Generator: Adobe Illustrator 18.1.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<svg version="1.1" id="Capa_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 viewBox="0 0 516 516" style="enable-background:new 0 0 516 516;" xml:space="preserve">
+<g>
+	<g>
+		<g>
+			<path style="fill:#010002;" d="M510,203c-2-8-8-13-16-14l-42-6c-2-6-1-3-4-9l26-34c5-6,5-15,1-22c0-1-11-18-35-42s-41-35-42-35
+				c-7-4-16-4-22,1l-34,26c-6-3-3-2-9-4l-6-42c-1-8-6-14-14-16c-1,0-21-6-55-6s-54,6-55,6c-8,2-13,8-14,16l-6,42c-6,2-3,1-9,4
+				l-34-26c-6-5-15-5-22-1c-1,0-18,11-42,35s-35,41-35,42c-4,7-4,16,1,22l26,34c-3,6-2,4-4,10l-42,5c-8,1-14,6-16,14c0,1-6,21-6,55
+				s6,54,6,55c2,8,8,13,16,14l42,6c2,6,1,3,4,9l-26,34c-5,6-5,15-1,22c0,1,11,18,35,42s41,35,42,35c7,4,16,4,22-1l34-26
+				c6,3,4,2,10,4l5,42c1,8,6,14,14,16c1,0,21,6,55,6s54-6,55-6c8-2,13-8,14-16l6-42c6-2,3-1,9-4l34,26c6,5,15,5,22,1
+				c1,0,18-11,42-35s35-41,35-42c4-7,4-16-1-22l-26-34c3-6,2-3,4-9l42-6c8-1,14-6,16-14c0-1,6-21,6-55S510,204,510,203z M475,291
+				l-40,5c-7,1-14,6-16,13c-6,17-4,12-11,27c-3,7-2,15,2,21l25,31c-5,6-12,15-22,25s-18,17-24,22l-32-25c-6-4-14-5-21-2
+				c-17,8-9,5-27,11c-7,2-12,9-13,16l-5,40c-8,1-19,2-33,2s-25-1-33-2l-5-40c-1-7-6-14-13-16c-17-6-12-4-27-11c-7-3-14-2-20,2
+				l-32,25c-6-5-15-12-25-22s-16-18-21-24l24-32c4-6,5-14,2-21c-8-15-5-10-11-27c-2-7-9-12-16-13l-40-5c-1-8-2-19-2-33s1-25,2-33
+				l40-5c7-1,14-6,16-13s5-13,5-14c0,0,3-7,6-13c3-7,2-14-2-20l-24-32c5-6,11-15,21-25s19-16,25-21l32,24c6,4,13,5,20,2
+				c15-8,10-5,27-11c7-2,12-9,13-16l5-40c8-1,19-2,33-2s25,1,33,2l5,40c1,7,6,14,13,16c17,6,12,4,27,11c7,3,15,2,21-2l32-24
+				c6,5,14,11,24,21s17,19,22,25l-25,32c-4,6-5,13-2,20c8,17,6,9,12,27c2,7,8,12,15,13l40,5c1,8,2,19,2,33S476,283,475,291z"/>
+			<path style="fill:#010002;" d="M258,135c-68,0-123,55-123,123s55,123,123,123s123-55,123-123S326,135,258,135z M258,342
+				c-47,0-84-37-84-84s37-84,84-84s84,37,84,84S305,342,258,342z"/>
+		</g>
+	</g>
+</g>
```

Comment:
They're part of the SVG, which was clearly auto-generated by some drawing program or other. It would be ill-advised to mess with this machine-generated code, so I don't think we should remove them.